### PR TITLE
Feature/update data handling part in receiving and updating keypair resource policy

### LINKED ISSUE #1508: Cannot see the keypair resource policy list clearly, since the value type of `allowed_vfolder_hosts` in keypair resource policy has been changed.
### Main idea

> NOTE: This feature is for admin users only.

Related Issue: [backend.ai#753](https://github.com/lablup/backend.ai/issues/753).

Because of the changes in Backend.AI Core mentioned above, we need to change the data handling part when fetching, creating, and updating keypair resource policy data.

### Alternative ideas

_No response_

### Anything else?

_No response_

### PR DESCRIPTION
**Pull request by lizable**
## Description
Related issue: [backend.ai#753](https://github.com/lablup/backend.ai/issues/753)
Related PR: [backend.ai#790](https://github.com/lablup/backend.ai/pull/790)

This PR resolves #1508.

### CODE REVIEW

File: src/components/backend-ai-resource-policy-list.ts [hunk 1]
```diff
@@ -596,6 +620,12 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
     const resourcePolicies = globalThis.backendaiclient.utils.gqlToObject(this.resourcePolicy, 'name');
     this.resource_policy_names = Object.keys(resourcePolicies);
     const resourcePolicy = resourcePolicies[policyName];
+    let allowedStorageHosts;
```
Comment (agatha197) [inline src/components/backend-ai-resource-policy-list.ts]: A blank exists at the end.

File: src/components/backend-ai-credential-view.ts [hunk 2]
```diff
@@ -375,14 +377,47 @@ export default class BackendAICredentialView extends BackendAIPage {
...
+   */
+  _getVfolderPermissions() {
+    globalThis.backendaiclient.storageproxy.getAllPermissions().then((res) => {
+      this.vfolderPermissions = res.vfolder_host_permission_list;
+    });
+  }
+
+  /**
+   * Parse simple allowed vfodler host list with fine-grained permissions
+   *
```
Comment (agatha197) [inline src/components/backend-ai-credential-view.ts]: please remove blank

File: src/components/backend-ai-credential-view.ts [hunk 3]
```diff
@@ -375,14 +377,47 @@ export default class BackendAICredentialView extends BackendAIPage {
...
+    globalThis.backendaiclient.storageproxy.getAllPermissions().then((res) => {
+      this.vfolderPermissions = res.vfolder_host_permission_list;
+    });
+  }
+
+  /**
+   * Parse simple allowed vfodler host list with fine-grained permissions
+   * 
+   * @param {Array<string>} storageList - storage list selected in `backend-ai-multi-select`
+   * @returns {Object<string, array>} - k-v object for storage host based permissions (all-allowed)
```
Comment (agatha197) [inline src/components/backend-ai-credential-view.ts:409]: please use @return instead.

File: src/components/backend-ai-resource-policy-list.ts [hunk 4]
```diff
@@ -702,9 +732,30 @@ export default class BackendAIResourcePolicyList extends BackendAIPage {
     return Math.round(gigabyte * value);
   }
 
+  /**
+   * Parse simple allowed vfodler host list with fine-grained permissions
+   * 
+   * @param {Array<string>} storageList - storage list selected in `backend-ai-multi-select`
+   * @returns {Object<string, array>} - k-v object for storage host based permissions (all-allowed)
```
Comment (agatha197) [inline src/components/backend-ai-resource-policy-list.ts]: same as above.

**Review by Sujin-Kim1 — CHANGES_REQUESTED** [review]
LGTM. But some lint errors exist.

File: src/components/backend-ai-credential-view.ts (hunk 3 above)
Comment (lizable) [reply src/components/backend-ai-credential-view.ts:409]: It seems like `@returns` is official. please check out the documentation below:
https://jsdoc.app/tags-returns.html

Comment (agatha197) [reply src/components/backend-ai-credential-view.ts:409]: oh, that's right. you don't need to change. I'm getting to know thanks to you.

**Review by Sujin-Kim1 — APPROVED** [review]
LGTM!

**Pull request merged**